src/core/Settings.js

```diff
@@ -658,6 +665,22 @@ import Events from './events/Events';
  * This value is used to specify the desired CMCD parameters. Parameters not included in this list are not reported.
  */
 
+/**
+ * @typedef {Object} module:Settings~CmsdSettings
+ * @property {boolean} [enabled=false]
+ * Enable or disable the CMSD response headers parsing.
+ * @property {module:Settings~CmsdAbrSettings} [abr]
+ * Sets additional ABR rules based on CMSD response headers.
+ */
+
+/**
+ * @typedef {Object} CmsdAbrSettings
+ * @property {boolean} [applyMb=false]
+ * Set to true if dash.js should apply CMSD maximum suggested bitrate in ABR logic.
+ * @property {number} [etpWeightRatio=0]
+ * Sets the weight ratio that shall be applied on CMSD estimated throuhgput compared to measured throughput when calculating throughput.
```

Comment:
We should add information here that this value is supposed to be between 0 and 1 with 
0 = 0%
1 = 100%